core-services/image-mirroring/openshift/mapping_origin_4_10

```diff
@@ -177,6 +177,7 @@ registry.ci.openshift.org/origin/4.10:local-storage-operator quay.io/openshift/o
 registry.ci.openshift.org/origin/4.10:local-storage-static-provisioner quay.io/openshift/origin-local-storage-static-provisioner:4.10 quay.io/openshift/origin-local-storage-static-provisioner:4.10.0 quay.io/openshift/origin-local-storage-static-provisioner:latest
 registry.ci.openshift.org/origin/4.10:loki quay.io/openshift/origin-loki:4.10 quay.io/openshift/origin-loki:4.10.0 quay.io/openshift/origin-loki:latest
 registry.ci.openshift.org/origin/4.10:machine-api-operator quay.io/openshift/origin-machine-api-operator:4.10 quay.io/openshift/origin-machine-api-operator:4.10.0 quay.io/openshift/origin-machine-api-operator:latest
+registry.ci.openshift.org/origin/4.10:machine-api-provider-openstack quay.io/openshift/origin-machine-api-provider-openstack:4.10 quay.io/openshift/origin-machine-api-provider-openstack:4.10.0 quay.io/openshift/origin-machine-api-provider-openstack:latest
```

Comment:
I assume other machine api providers will show up using this pattern in the future?  Will it be all providers having their own image?

Reply:
I think @JoelSpeed or @Fedosin can confirm, but I believe that's consistent with the pattern we're going for.

Reply:
Its likely that will be the case

Reply:
Yes, we are intending to move all of the existing machine api providers to this format during the 4.11 timeframe